//-------------------------------------------------------------------------------
// Name:        nmea_server.proto
// Purpose:     protobuf and grpc for navigation server data streams
//
// Author:      Laurent Carré
//
// Created:     12/06/2022
// Copyright:   (c) Laurent Carré Sterwen Technology 2021-2023
// Licence:     Eclipse Public License 2.0
//-------------------------------------------------------------------------------

syntax = "proto3";

import "nmea_messages.proto";

service NMEAServer {
  rpc status (server_cmd) returns (server_resp) {}
  rpc sendNMEA( nmea_msg ) returns (server_resp) {}
  rpc getNMEA(server_cmd) returns (stream nmea_msg) {}
}

